ui/k-info-panel/switch_info.kytos

```diff
@@ -173,6 +180,34 @@
        request.done(this.msg_state_success)
        request.fail(this.msg_state_failure)
      },
+     bt_request(){
+       var temp = {}
+       temp["interfaces"] = this.chosen_interface
+       let request = $.ajax({
+                        type: "POST",
+                        url: this.$kytos_server_api + "kytos/of_lldp/v1/interfaces/" + this.chosen_option.toLowerCase(),
+                        async: true,
+                        contentType: "application/json",
+                        data: JSON.stringify(temp),
+       });
+       var _this = this
+       request.done(function(data){
+         let notification = {
+           title: 'Interfaces ' + _this.chosen_option + 'd: Success',
```

Comment:
Could you add LLDP to this string something link `Interfaces LLDP`? I think it would give the full semantics of the operations, otherwise it'll show up as `Interfaces Enabled: Success`, which might confuse some operators since they're enabling LLDP on that interface and not the interface, see what I mean?

Reply:
Updated on commit [a48b869d69b914523a9b9bd606c8cdc74059e42e](https://github.com/kytos-ng/topology/pull/81/commits/a48b869d69b914523a9b9bd606c8cdc74059e42e)